lib/Traits/CatalogMemberTraits.ts

```diff
@@ -39,6 +39,13 @@ export class ShortReportTraits extends ModelTraits {
     description: "The content of the section."
   })
   content?: string;
+
```

Comment:
This is already in `next` not sure why it is showing a diff

Reply:
merge in `next` and see if it disappears?